src/index.html

```diff
@@ -1,14 +1,78 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#page-top" class="header__logo--link">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li  class="nav__item ">
+            <a href="#apple" class="nav__item--link is-active">
+            Apple
+            </a>
```

Comment:
```suggestion
            <a href="#apple" class="nav__item is-active">
              Apple
            </a>
```
The same, and don't forget about formatting

Reply:
hi, fixed